helloworld-consumer.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: extensions/v1beta1
```

Comment:
`deployment.yaml` contains `helloworld-consumer.yaml` and `helloworld-producer.yaml` files because it is deploying consumer and producer. Do you have any other suggestion?